f5router/bigipResources/bigipResources.go

```diff
@@ -93,7 +93,9 @@ type (
 	// Condition for a rule
 	Condition struct {
 		Equals      bool     `json:"equals,omitempty"`
+		BeginsWith  bool     `json:"beginsWith,omitempty"`
```

Comment:
We should be sure to manually test this in a CF environment to be certain it creates the proper rule. Right now as CCCL is maturing I want to be sure it supports any additional functionality we add.

Reply:
That is a very good idea.